Tasks/UseRubyVersion/task.loc.json

```diff
@@ -0,0 +1,64 @@
+{
+  "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
+  "name": "UseRubyVersion",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Tool",
+  "runsOn": [
+    "Agent",
+    "DeploymentGroup"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 133,
+    "Patch": 0
+  },
+  "preview": true,
+  "demands": [],
+  "instanceNameFormat": "ms-resource:loc.instanceNameFormat",
+  "inputs": [
+    {
+      "name": "versionSpec",
+      "type": "string",
+      "label": "ms-resource:loc.input.label.versionSpec",
+      "required": true,
+      "defaultValue": ">= 2.4",
+      "helpMarkDown": "ms-resource:loc.input.help.versionSpec"
+    },
+    {
+      "name": "outputVariable",
+      "type": "string",
+      "label": "ms-resource:loc.input.label.outputVariable",
+      "required": false,
+      "defaultValue": "",
+      "helpMarkDown": "ms-resource:loc.input.help.outputVariable"
+    },
+    {
+      "name": "addToPath",
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.addToPath",
+      "required": false,
+      "defaultValue": "true",
+      "helpMarkDown": "ms-resource:loc.input.help.addToPath"
+    }
+  ],
+  "outputVariables": [
+    {
+      "name": "outputVariable",
```

Comment:
This name will be visible to the user, so it should probably be something like `rubyLocation`